packages/ts/src/containers/xy-container/index.ts

```diff
@@ -50,6 +46,7 @@ export class XYContainer<Datum> extends ContainerCore {
   private _clipPathId = guid()
   private _axisMargin: Spacing = { top: 0, bottom: 0, left: 0, right: 0 }
   private _firstRender = true
+  private xyDomain = new Map<ScaleDimension, number[]>()
```

Comment:
We prefix all private variables with underscore `_`. The rule is not enforced with ESLint yet, but we follow it.